test/blocks/global-navigation/global-navigation.test.js

```diff
@@ -1161,4 +1199,164 @@ describe('global navigation', () => {
       expect(getOverflowingTopnav()).to.equal(null);
     });
   });
+
+  describe('Universal navigation', () => {
+    let unavMeta;
+    const orgAlloy = window.alloy;
+    beforeEach(async () => {
+      window.UniversalNav = sinon.spy();
+      // eslint-disable-next-line no-underscore-dangle
+      window._satellite = { track: sinon.spy() };
+      // eslint-disable-next-line
+      window.alloy = () => new Promise((resolve) => resolve({ identity: { ECID: 'dummy-ECID' } }));
+      unavMeta = toFragment`<meta name="universal-nav" content="on">`;
+      document.head.append(unavMeta);
+    });
+
+    afterEach(() => {
+      sinon.restore();
+      window.allow = orgAlloy;
+      if (document.head.contains(unavMeta)) document.head.removeChild(unavMeta);
+    });
+
+    describe('desktop', () => {
+      it('should render the Universal navigation', async () => {
+        const fullUnavMeta = toFragment`<meta name="universal-nav" content="profile, appswitcher, notifications, help">`;
+        const gnav = await createFullGlobalNavigation();
+        await gnav.decorateUniversalNav();
```

Comment:
Should this be moved to the `createFullGlobalNavigation` helper, with a param to trigger decorating the universal nav or the old one?

Reply:
Agree on this one too, we might need to decorate the UNav in other tests too, let's have it available anywhere through `createFullGlobalNavigation`

Reply:
Good suggestion! Made the change 👍